modules/boilerplate/globals.js

```diff
@@ -90,6 +90,16 @@ setGlobal('buildUrl', (sectionName, pageName) => {
     }
 });
 
+setGlobal('createHeroImage', function(opts) {
```

Comment:
I'm passing named parameters through rather than straight passing `opts` as I prefer the explicitness.